sql/stripe_derived/charge_events/view.sql

```diff
@@ -0,0 +1,31 @@
+CREATE OR REPLACE VIEW
+  charge_events
+AS
+WITH raw AS (
+  SELECT
+    created AS event_timestamp,
+    `data`.charge.*,
+  FROM
+    stripe_external.events_v1
+  WHERE
+    `data`.charge IS NOT NULL
+  UNION ALL
+  SELECT
+    created AS event_timestamp,
+    *,
+  FROM
+    stripe_external.charges_v1
+  -- TODO remove this filter in favor of deleting the unwanted data from the table
+  WHERE
+    DATE(created) < DATE '2020-08-10'
+)
+SELECT
```

Comment:
Are we intentionally keeping customer IDs and subscription IDs out of this? If not, shouldn't we include them here so we can e.g. join to subscriptions?

Reply:
I'm going to remove charge_events for now, as it's not being used within this PR